CHANGELOG.md

```diff
@@ -13,3 +13,4 @@ This changelog only contains the changes that are unreleased. For changes for in
 - Update gradle wrapper version from 8.2 to 8.7 [#886]
 - Migrate to the new Gradle version catalogs for libraries and plugins
 - Update the `application.yml` GitHub workflow [#889]
+- Migrate to Gradle KTS [#898](https://github.com/ATLauncher/ATLauncher/pull/898)
```

Comment:
No need to link to the url, just the issue number in square braces is fine, the whole link would likely break the release process

```suggestion
- Migrate to Gradle KTS [#898]
```